packages/tessellator/lib/tessellator.dart

```diff
@@ -0,0 +1,247 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Lets not make this a separate package? It will be 3 packages is easier to publish than 4, and I don't know where we would use this outside of the compiler

Reply:
I did this purely for testing purposes. This patch is no where near ready to land yet. I literally copy-pasted this from the engine code because consuming it from there is harder to share